includes/polling/functions.inc.php

```diff
@@ -199,10 +199,10 @@ function record_sensor_data($device, $all_sensors)
         // FIXME also warn when crossing WARN level!
         if ($sensor['sensor_limit_low'] != '' && $prev_sensor_value > $sensor['sensor_limit_low'] && $sensor_value < $sensor['sensor_limit_low'] && $sensor['sensor_alert'] == 1) {
             echo 'Alerting for '.$device['hostname'].' '.$sensor['sensor_descr']."\n";
-            log_event("$class {$sensor['sensor_descr']} under threshold: $sensor_value $unit (< {$sensor['sensor_limit_low']} $unit)", $device, $class, 4, $sensor['sensor_id']);
```

Comment:
Why did you remove this?

Reply:
i removed from log event `'$sensor['sensor_descr']'`, because it's used in `type` so sensor_type can be seen now there.

Reply:
```
Voltage vddgfx under threshold: 0.725 (< 0.8075 )
Voltage Vcore above threshold: 1.072 (> 1.0488 )
```

If we remove the second word how can we know which voltage this is for example?

Reply:
I don't think you can remove it.

Reply:
You could maybe try to remove text duplication if that what you mean.
Like:
```php
$descr = $class == $sensor['sensor_descr'] ? $class : "$class {$sensor['sensor_descr']}";
```